assets/RFC003-EthereumBitcoin-AtomicSwap.puml

```diff
@@ -0,0 +1,132 @@
+' DIAGRAM #########################################
+' RFC003 atomic swap
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Alice
+participant "Alice\nComit-Node" as AC #lightgreen
+participant "Alice\nETH-Client" as AETH #lightcyan
```

Comment:
I would skip these participants and only include the `Ethereum Network` (or `Beta Ledger`)

Reply:
I think it has to be important to understand that the users have to communicate with a blockchain client to get their transactions on the network. I would not remove this. 
I feel the "network" is too abstract if it stands alone.

Reply:
In the end I changed it to Alpha and Beta Ledger as it should depict the RFC.